src/ACEtk/details/verify/src/sorted.hpp

```diff
@@ -0,0 +1,40 @@
+template< typename Range >
+inline bool sorted( Range& array ){
+
+  auto sortedEnd = ranges::is_sorted_until( array );
+  if( sortedEnd != array.end() ){
+    njoy::Log::error( "array is not sorted");
+    njoy::Log::info( "first non-sorted element is: {} at index: {}", 
```

Comment:
if you are going to log, please show the unsorted neigbor as well

Reply:
👍